src/user/using-the-capture-app.md

```diff
@@ -872,6 +872,14 @@ On the enrollment dashboard, you can view the Person Profile widget.
 
 The Person profile widget is used for viewing key attributes but not for editing. In order to edit the person profile, you must select the `Edit` button to open an edit window. 
 
+### Warning widget
+On the enrollment dashboard, the warning widget will be displayed.
```

Comment:
I suggest removing this line and incorporating it into the sentence below, see next suggestion.